### DISCUSSION TITLE
How do I enable fast cursor blink rate

### DISCUSSION THREAD

**Discussion by MikePresman** [wezterm/wezterm#3389] [Q&A] [answered]
I've tried

```
  default_cursor_style = "BlinkingBar",
  cursor_blink_rate = 0.1,
```
and with cursor_blink_rate set to like 1000

but neither of them gets me a fast blink rate like we see when we type in this chat here !.
I dislike the easing out and easing in, I just would like rapid cursor blinking like normal.

Thanks !

**Comment by yutkat** [comment, marked as answer]
The cursor_blink_rate should be set in milliseconds. The following modification worked fine for me.

https://wezfurlong.org/wezterm/config/lua/config/cursor_blink_rate.html?h=cursor_blink_rate

```lua
default_cursor_style = "BlinkingBar",
cursor_blink_rate = 100,
```

If it doesn't work, try this minimum configuration.

```lua
-- /tmp/wezterm.lua
return {
	default_cursor_style = "BlinkingBar",
	cursor_blink_rate = 100,
}
```

`wezterm --config-file /tmp/wezterm.lua start`

**Reply by wez** [reply]
In addition, if you don't like easing, you can set it to step blink:

```lua
config.cursor_blink_ease_in = 'Constant'
config.cursor_blink_ease_out = 'Constant'
```

**Reply by MikePresman** [reply]
The cursor_blink_ease_in and cursor_blink_ease_out was also crucial. Thanks to both of you.
